packages/cli-e2e/utils/browser.ts

```diff
@@ -32,9 +34,13 @@ export async function closeAllPages(browser: Browser) {
 
 async function getWsUrl(): Promise<string> {
   const chromeDebugInfoRaw = (
-    await axios.get<JsonVersionFile>(CHROME_JSON_DEBUG_URL)
+    await axios.get<JsonVersionFile>(CHROME_JSON_DEBUG_URL, {
+      httpAgent: new HttpAgent({family: 4}),
+    })
   ).data;
-  return chromeDebugInfoRaw.webSocketDebuggerUrl;
+  const wsUrl = new URL(chromeDebugInfoRaw.webSocketDebuggerUrl);
+  wsUrl.hostname = (await dnsPromises.resolve4(wsUrl.hostname))[0];
+  return wsUrl.toString();
```

Comment:
Node 18 use ipv6 by default for pretty much everything, pupeeter doesn't like ipv6.